resource-managers/kubernetes/docker/src/main/dockerfiles/spark/Dockerfile

```diff
@@ -18,6 +18,7 @@
 FROM openjdk:8-alpine
 
 ARG spark_jars=jars
+ARG example_jars=examples/jars
```

Comment:
could we make this optional? if someone wants to build a smaller image without example

Reply:
Different change? ATM the existing code does not make this optional.

Reply:
Also the K8S integration tests currently assume that the examples JAR is present since they rely upon running code from the examples